src/components/SiteSettingsMenu.tsx

```diff
@@ -139,14 +141,55 @@ export const SUPPORTED_NETWORKS: {
     rpcUrls: ["https://rpc.ftm.tools"],
     blockExplorerUrls: ["https://ftmscan.com"],
   },
+  [ChainId.EVMOS]: {
+    chainId: hexlify(9001),
+    chainName: "Evmos",
+    nativeCurrency: {
+      name: "Evmos",
+      symbol: "EVMOS",
+      decimals: 18,
+    },
+    rpcUrls: ["https://eth.bd.evmos.org:8545"],
+    blockExplorerUrls: ["https://evm.evmos.org"],
+  },
 }
+
+export const DEV_SUPPORTED_NETWORKS: {
```

Comment:
I wonder if it's worth just adding a "isTestnet" flag on the exising SUPPORTing_NETWORKS type instead of creating a new type